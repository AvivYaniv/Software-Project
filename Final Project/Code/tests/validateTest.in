solve tests/b2_2-notValid.txt
print_board
validate
solve tests/b2_2.txt
validate
set 2 2 1
validate
edit tests/b2_2-notValid.txt
validate
edit tests/b2_2.txt
validate
exit

