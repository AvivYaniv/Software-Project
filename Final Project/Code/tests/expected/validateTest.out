Sudoku
------
Enter your command:
-------------------
|  1.  2.|  4.    |
|        |  3.  1.|
-------------------
|        |      2.|
|        |        |
-------------------
Enter your command:
-------------------
|  1.  2.|  4.    |
|        |  3.  1.|
-------------------
|        |      2.|
|        |        |
-------------------
Enter your command:
Validation failed: board is unsolvable
Enter your command:
-------------------
|  1.  2.|  4.    |
|        |      1.|
-------------------
|        |      2.|
|        |        |
-------------------
Enter your command:
Validation passed: board is solvable
Enter your command:
-------------------
|  1.  2.|  4.    |
|      1*|      1.|
-------------------
|        |      2.|
|        |        |
-------------------
Enter your command:
Error: board contains erroneous values
Enter your command:
-------------------
|  1   2 |  4     |
|        |  3   1 |
-------------------
|        |      2 |
|        |        |
-------------------
Enter your command:
Validation failed: board is unsolvable
Enter your command:
-------------------
|  1   2 |  4     |
|        |      1 |
-------------------
|        |      2 |
|        |        |
-------------------
Enter your command:
Validation passed: board is solvable
Enter your command:
Exiting...
